src/routes/(admin)/questionbank/single-question/+page.svelte

```
@@ -0,0 +1,182 @@
+										<GripVertical class="my-auto h-full  rounded-sm bg-gray-100" />
+										<Input class=" border-0" name={key} bind:value={totalOptions[index].value} />
+									</div>
+									<div class="flex flex-row gap-2">
+										<Checkbox
+											onCheckedChange={(checked: boolean) =>
+												(totalOptions[index].correct_answer = checked)}
+										/><Label class="text-sm ">Set as correct answer</Label>
+									</div>
+								</div>
+							</div>
+						{/each}
```

ESLint

[error] Each block should have a key

(svelte/require-each-key)